ConnorAndersonLarson/PlanetParty
# Style/responsiveness sort box

# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?


### Are there any known bugs?


### Summary of the changes
Add responsive styling to all planets view and sort container. Added icons and removed radio button default styling.


### What ticket(s) do the changes resolve?

#32 
Please Link Issues to this PR as necessary.

### Notes for the Reviewer
Only slightly tweaked the single planet view, didn't want to start adding responsive styling to this page without actual data being displayed, and didn't know what the data would look like. Also the responsive design needs a second opinion, and the breakpoints for media queries may need change.


### How to Test Changes?

Open site local, shrink and expand the homepage/all planets view. Site styles should change with screen size.

### Next Iterations

Add responsive styling to single planet view. Figure out proper text size for the entire website along with colors. Maybe refactor/add colors and/or icons to the sort icons
